.gitlab-ci.yml

```diff
@@ -0,0 +1,124 @@
+image: golang:1.11-alpine3.9
+
+cache:
+  paths:
+    - /go/src/github.com
+    - /go/src/golang.org
+    - /go/src/google.golang.org
+    - /go/src/gopkg.in
+
+variables:
+   DOCKER_HOST: tcp://docker:2375/
+   DOCKER_DRIVER: overlay2
+   DOCKER_REPOSITORY: mendersoftware/mender-artifact
+
+services:
+  - docker:dind
+
+stages:
+  - test
+  - build
+  - publish
+  - integration
+
+before_script:
```

Comment:
This `before_scripts` seems to be used only in `test:static`.

I suggest: making it generic so that it can be reused or moving it directly into test:static (and then refactor a bit the other steps so that they have a proper before_script as well)